amgut/lib/test/test_util.py

```diff
@@ -0,0 +1,30 @@
+from unittest import TestCase, main
+from amgut.lib.util import survey_fermented, survey_surf
+
+
+class TestUtil(TestCase):
+    def test_survey_fermented(self):
```

Comment:
Can you add tests to `survey_vioscreen` and `survey_asd` given that you modified them?